WeScan/Edit/EditScanViewController.swift

```diff
@@ -191,12 +191,12 @@ final class EditScanViewController: UIViewController {
         quadViewHeightConstraint.constant = frame.size.height
     }
     
-    /// Generates a `Quadrilateral` object that's centered and one third of the size of the passed in image.
+    /// Generates a `Quadrilateral` object that's centered and 90% of the size of the passed in image.
     private static func defaultQuad(forImage image: UIImage) -> Quadrilateral {
-        let topLeft = CGPoint(x: image.size.width / 3.0, y: image.size.height / 3.0)
-        let topRight = CGPoint(x: 2.0 * image.size.width / 3.0, y: image.size.height / 3.0)
-        let bottomRight = CGPoint(x: 2.0 * image.size.width / 3.0, y: 2.0 * image.size.height / 3.0)
-        let bottomLeft = CGPoint(x: image.size.width / 3.0, y: 2.0 * image.size.height / 3.0)
+        let topLeft = CGPoint(x: image.size.width / 20.0, y: image.size.height / 20.0)
+        let topRight = CGPoint(x: 19.0 * image.size.width / 20.0, y: image.size.height / 20.0)
+        let bottomRight = CGPoint(x: 19.0 * image.size.width / 20.0, y: 19.0 * image.size.height / 20.0)
+        let bottomLeft = CGPoint(x: image.size.width / 20.0, y: 19.0 * image.size.height / 20.0)
```

Comment:
Can you explain how deviding by 20.0 and something 19.0 results in 90% of the size?

Reply:
@AvdLee Thank you for the question. Let me elaborate.
Let's assume `image.size.width` is equal to `100.0` and diving it by 20.0 gives us 5.0 and multiplying it by 19.0 gives us 95; resulting so called quad width to be 90.0. 5% on each edge results 10% on each axis.

Reply:
On the other hand previous code states that 

> one third of the size of the passed in image

However it was 1/9 of the size of the passed in image not 1/3

Reply:
Wouldn't it be a lot simpler and more readable to do * 0.9? 

E.g. 100 * 0.9 = 90

Reply:
Yeah, it could be. 
I can make following changes if you like:
- `image.size.width / 20.0` to `image.size.width * 0.05`
- `19.0 * image.size.width / 20.0` to `image.size.width * 0.95`

Reply:
Yes, that would make a lot more sense to me!

Reply:
Done ✅

Reply:
🚀